src/gui/gui_event.cpp

```diff
@@ -108,8 +108,8 @@ void populate_event_submap(sys::state& state, text::substitution_map& sub,
 	text::add_to_substitution_map(sub, text::variable_type::capital, target_capital);
 	text::add_to_substitution_map(sub, text::variable_type::monarchtitle, state.culture_definitions.governments[state.world.nation_get_government_type(target_nation)].ruler_name);
 	// TODO: Is this correct? I remember in vanilla it could vary
-	text::add_to_substitution_map(sub, text::variable_type::culture, state.world.culture_get_name(state.world.nation_get_primary_culture(target_nation)));
-	text::add_to_substitution_map(sub, text::variable_type::culture_group_union, state.world.culture_get_name(state.world.nation_get_primary_culture(target_nation)));
+	text::add_to_substitution_map(sub, text::variable_type::culture, text::produce_simple_string(state, state.world.culture_get_name(state.world.nation_get_primary_culture(target_nation))));
```

Comment:
you can't add temporary strings to substitution maps, or they will end up as dangling references